app/models/miq_provision_virt_workflow.rb

```diff
@@ -119,8 +119,7 @@ def set_on_vm_id_changed
        :placement_rp_name,
        :linked_clone,
        :snapshot,
-       :placement_dc_name,
-       :placement_storage_profile]
```

Comment:
```placement_storage_profile``` is the first case where the value can be pre-selected without user's selection when the selected template is attached with a storage profile.